### ISSUE TITLE
Require an Old version of Crowdfunding for Joomla 3.0 

### ISSUE THREAD

**Issue by bharatthakkar** [ITPrism/CrowdFunding#159]
Anybody who have old copy of  version of Crowdfunding which can work with Joomla 3.0, Please if possible share with me....

**Comment by ITPrism** [comment]
Hi,
You can download it from here.
http://cdn.itprism.com/extensions/crowdfunding/pkg_crowdfunding_v1.11-J3.3.zip

**Comment by bharatthakkar** [comment]
This version is also not working... I require Version which works with Joomla 3.0 and not 3.1, 3.2 or 3.3

**Comment by ITPrism** [comment]
Hmm...First version for Joomla! 3 was developed over v3.1.
I guess, you can use it on 3.0.
Here you are a link.
http://cdn.itprism.com/extensions/crowdfunding/pkg_crowdfunding_v1.4.3-J3.1.zip

**Comment by bharatthakkar** [comment]
Their is no such file available on above link... it gives Error 403....

**Comment by ITPrism** [comment]
ops :)
Try it now.

**Comment by bharatthakkar** [comment]
Thankyou it works.....

BUT....

Dashboard is not coming - See ScreenShot Below...

![screen01](https://cloud.githubusercontent.com/assets/4311399/8523061/b955a8dc-2410-11e5-9af9-75ad5498c099.jpg)

Projects Gives Error
 JHtmlGrid::checkall not found. 

![screen02](https://cloud.githubusercontent.com/assets/4311399/8523164/4407ab88-2411-11e5-809e-1e4fa948d23e.jpg)

Same Error occurs in all other options also... i.e. in Transaction, Locations, Countries ... every...

**Comment by bharatthakkar** [comment]
So, I think on Yahoo Host - Crowdfunding Extension is not my baby.....

**Comment by ITPrism** [comment]
:)
I think, [SiteGround] (https://www.siteground.com/) will be better option.
You can try it here https://www.joomla.com/

**Comment by bharatthakkar** [comment]
Yeah, Dear I am aware with this... Not only joomla.com, But following 40 other sites which provides free hosting and I am already registered there. My team is developing some or other testing things on those free hosting websites. My Problem is I am already having two Domains "bgtinfosolutions.com" and "ngocsr.com" are hosted on Yahoo hosting, Paid for 10 years, which has unlimited space, database, unlimited data transfer, 1000 sub-domains in each and 99.99% uptime. (In fact I have 5 such domains on Yahoo host, but other 3 are for some specific NGOs)
And the Constraints are : Yahoo has
a.	PHP 5.3.6
b.	Perl 5.8.7
c.	MySQL 5.1 
And I have to work with this constraints only.

My Need is - on my Domains "www.ngocsr.com" and "www.terofoundation.com" we are working with about 200 Non-Profit-Organisations (NPO/NGO). They are working with me on many different Projects. I decided to provide them free crowdfunding service on my Website for their projects and as I am Joomla Fan Since the beginning (My all websites are Joomla based) So I concluded on this Extension. But anyway is not my baby..... 

On Joomla.com or on any such other free hosting I cannot put a large data of my all NGOs, as these websites provides limited hosting space. But I will try for some.

Thanks for your Supporting conversation always.     

Following are other 40 free hosting sites on which we are working. We have 148 other also, but all have limited space. 
CFSites.org
Doodlekit
Yola 
Weebly
WordPress.com
WebsiteBuilder
50 Webs
Free Web Hosting Area
ByetHost
FreeHostingEU
BIZ.ht
BIZ.nf 
WIX
000webhost.com
EDUBlogs
WEBS
5GBfree
Jimdo
Zymic
1FreeHosting
FreeHostingNoAds
AgilityHoster
ZettaHost
x10hosting
hPage
Tripod/Lycos
Sitey
Angelfire/Lycos
WebNode
IM Creator
WebStarts
Moonfruit
Ezweb123
Doomby
SnapPages
Edicy
uCoz
SJuga
0Fees
Get Social Shops
